text/chapter1.md

```diff
@@ -115,7 +115,7 @@ main = log "Hello, World!"
 Commands which should be typed at the command line will be preceded by a dollar symbol:
 
 ```text
-$ spago build
+spago build
```

Comment:
The dollar symbols here are intentional. But I'm not against removing that pattern from the book.

Reply:
You are correct. It would be better to unify with the existing notation. I reverted the changes and disabled the rule.